SRC/classes/AccountInsertRecord.cls

```diff
@@ -0,0 +1,33 @@
+/**
+*  Description     :  Insert Record IN Account
+*
+*  Created By      :  Kamlesh Kumar
+*
+*  Created Date    :  07/04/2022
+*
+*  Revision Logs   :    V1.0 - Created - Kamlesh Kumar
+**/
+public with sharing class AccountInsertRecord {
+    /**
+    *   @description    :  Insert Record IN Account
+    *
+    *   @return         :   void
+    *
+    *   @revision Log   :   V1.1 - Created  - 07/04/2022 - Developer Name - Kamlesh Kumar
+    **/
+    public AccountInsertRecord() {
+        Account acc = new Account();
+        acc.Name = 'Shahil';
+        acc.AccountNumber = '324964587673';
+        acc.Phone = '5125562355';
+        System.debug(acc);
+        insert acc;
+        List<Account> acc_4 = [SELECT Name FROM Account Where Name = 'Shahil'];
+        for(Account accName : acc_4) {
+            if(accName.Name.equals('Shahil')) {
+                accName.Name = 'Aron';
+            }
+        }
+        update acc;
```

Comment:
False update, Not going to approve the PR if one more time you performed the changes 
Check and apply debug logs, is you account updated because you haven't assigned the values to acc while updating it